docs/product/explore/session-replay/index.mdx

```diff
@@ -4,9 +4,13 @@ sidebar_order: 70
 description: "Use Session Replay to get reproductions of user sessions to improve your app experience."
 ---
 
-Sentry has Session Replay for both Web and Mobile. Both are generally available and stable.
+Sentry supports Session Replay for Web and Mobile. This includes browser-based applications and certain native mobile platforms, such as Android, iOS, and React Native. Both versions of Replay (Web and Mobile) are generally available and stable.
 
-Learn more here:
+For browser-based applications (Web replay), this includes static websites, single-page applications, and also server-side rendered applications. This includes platforms such as Electron, Next.js, and Remix. The only prerequisite is that your application uses Sentry JavaScript SDK (version 7.2.0 or greater) either with NPM/Yarn or with our JS Loader script.
```

Comment:
```suggestion
For browser-based applications (Web replay), this includes static websites, single-page applications, and also server-side rendered applications. For example, platforms such as Electron, Next.js, and Remix. The only prerequisite is that your application uses Sentry JavaScript SDK (version 7.2.0 or greater) either with NPM/Yarn or with our JS Loader script.
```

Reply:
the pre-requisite for SDK version seems out of place - unless we also add the same note for Mobile imo